core/croquet/src/main/java/org/lgna/croquet/PredeterminedMenuModel.java

```diff
@@ -53,11 +53,15 @@
 public abstract class PredeterminedMenuModel extends StaticMenuModel {
   private final StandardMenuItemPrepModel[] models;
 
-  public PredeterminedMenuModel(UUID individualId, StandardMenuItemPrepModel... models) {
-    super(individualId);
+  public PredeterminedMenuModel(UUID individualId, boolean showScrollArrows, StandardMenuItemPrepModel... models) {
```

Comment:
I am not a fan of booleans like the `showScrollArrows` boolean as part of the constructor. It does not change within  a class so it could be done as a checking method that is only overridden when they are to be shown.
Not sure I am asking for you to change this, since my distaste for the menu class hierarchy could lead me to suggest many more changes.
Something to consider before adding such flags in the future?